dbtmetabase/models/config.py

```diff
@@ -0,0 +1,30 @@
+from dataclasses import dataclass, field
+from typing import Optional, Iterable, Union
+
+
+@dataclass
+class MetabaseConfig:
+    # Metabase Client
+    metabase_database: str
+    metabase_host: str
+    metabase_user: str
+    metabase_password: str
+    # Metabase additional connection opts
+    metabase_use_http: bool = False
+    metabase_verify: Union[str, bool] = True
+    # Metabase Sync
+    metabase_sync_skip: bool = False
+    metabase_sync_timeout: Optional[int] = None
+
+
+@dataclass
+class dbtConfig:
```

Comment:
This should be `DbtConfig`. I know dbt themselves sometimes use lower case, but that's against Python style guidelines and we already have `DbtFolderReader` etc, so I prefer `DbtConfig`.

Reply:
Changed to `DbtConfig` to conform to style guidlines